### PR TITLE
Feature/add active version in version listing

### PR DESCRIPTION
**Pull request by guilhermocc**
## What ❓ 
Add active version field in scheduler version listing.

## Why 🤔 
Currently, this endpoint does not inform what is the active scheduler, since now an active scheduler can be any version, we need to include this information in the endpoint response.

## How
- Update SchedulerStorage postgres adapter
- Update ListScheduler message in proto definition
- Update SchedulerHandler and tests

### CODE REVIEW

**Review by caiordjesus — APPROVED** [review]
LGTM

File: internal/adapters/scheduler_storage/pg/scheduler_version.go [hunk 1]
```diff
@@ -29,12 +29,14 @@ import (
 
 type SchedulerVersion struct {
 	Version   string      `db:"version"`
+	IsActive  bool        `db:"is_active"`
```
Comment (lyracampos) [inline internal/adapters/scheduler_storage/pg/scheduler_version.go:32]: @guilhermocc IsActive(is_active) is just a logical property, so we won't have a column in the database to represent it, right?

**Review by lyracampos — APPROVED** [review]

**Comment by lyracampos** [comment]
👍

File: internal/adapters/scheduler_storage/pg/scheduler_version.go (hunk 1 above)
Comment (guilhermocc) [reply internal/adapters/scheduler_storage/pg/scheduler_version.go:32]: exactly! in the query, we use an alias `is_active` for this calculated query result, so I had to add this annotation here

**Pull request merged**